Source/WebKit/GPUProcess/graphics/RemoteDisplayListRecorder.cpp

```diff
@@ -318,11 +318,11 @@ void RemoteDisplayListRecorder::drawNativeImageWithQualifiedIdentifier(Qualified
     handleItem(DisplayList::DrawNativeImage(imageIdentifier.object(), imageSize, destRect, srcRect, options), *image);
 }
 
-void RemoteDisplayListRecorder::drawSystemImage(SystemImage& systemImage, const FloatRect& destinationRect)
+void RemoteDisplayListRecorder::drawSystemImage(Ref<SystemImage>&& systemImage, const FloatRect& destinationRect)
```

Comment:
I don't know if it makes a difference, but:

I think for Ref/RefPtr and similar, it's more performant to pass in by value.
Contrast:
The caller does not know whether the callee moves the object off. So caller must provide a ptr pointing to their stack. After the call, caller must destruct the object.

VS.

Te caller knows that they don't need the object. They can construct the object directly into the conceptual callee argument stack.

To visualize it the another way:

consider
```
void f() {
     auto si = getSystemImage();
     drawSystemImage(WTFMove(si);
}
```

The call is in tail-call position. If the drawSystemImage takes the value, the compiler could omit the return to f() and instead return to the caller of f().

If the drawSystemImage takes rvalue ref, the return must happen to f() in order to do the maintainance for possibly non-moved Ref<SystemImage> si.

Reply:
I see, adopted.